components/Sponsors.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { H1, Section } from './style'
+import { Style as Style_LogoGrid } from './LogoGrid'
+import { Row, Col } from './style'
+
+const logos = [
+    // Row 1
+    'Point72.jpg',
+    'Bendheim.png',
+]
+
+export const SponsorComp = () => {
```

Comment:
The convention is 2-space tabs.  Did you install the editor extensions mentioned in the README.md?  That will fix the style issues.